docs/administration/vulnerability-scanning/scan-individual-artifact.md

```diff
@@ -0,0 +1,69 @@
+
+    **NOTE**: You can start a scan at any time, unless the status is **Queued** or **Scanning**. If the database has not been fully populated, you should not run a scan. The following statuses are displayed in the **Vulnerabilities** column:
+
+    * **Not Scanned:** The artifact has never been scanned.
+    * **Unsupported:** The artifact is not supported by the scanner.
+    * **Queued:** The scanning task is scheduled but has not run yet.
+    * **Scanning:** The scanning task is in progress and a progress bar is displayed.
+    * **View log:** The scanning task failed to complete. Click **View Log** link to view the related logs.
+    * **Complete:** The scanning task completed successfully.
+
+    If the process completes successfully, the result indicates the overall severity level, with the total number of vulnerabilities found for each severity level, and the number of fixable vulnerabilities.
+
+    ![Scan result](../../../img/scan-result.png)
+
+    * **Red:** At least one critical vulnerability found
+    * **Orange:** At least one high level vulnerability found
+    * **Yellow:** At least one medium level vulnerability found
+    * **Blue:** At least one low level vulnerability found
+    * **Green:** No vulnerabilities found
+    * **Grey:** Unknown vulnerabilities
+
+1. Hover over the number of fixable vulnerabilities to see a summary of the vulnerability report.
+
+    ![Vulnerability summary](../../../img/vulnerability-summary.png)
+
+1. Click on the artifact digest to see a detailed vulnerability report.
+
+    ![Vulnerability report](../../../img/artifact-detail.png)
+
+    In addition to information about the artifact, all of the vulnerabilities found in the last scan are listed. You can order or filter the list by the different columns. You can also click **Scan** in the report page to run a scan on this artifact.
+
+
+## Vulnerability scanning for OCI image index
+
+From the version 2.0, Harbor can scan OCI image index. The scanner has the capacity to scan the OCI image index when the scanner has the capacity to scan one of the artifacts referenced by the OCI image index.
+
+When scan the OCI image index, Harbor will send scan requests for each of the artifact which is supported by the scanner and referenced by the OCI image index to the scanner. If one of the process is **Scanning**, the process for the OCI image index is **Scanning**. After all the processes are completed, the process for the OCI image index is successfully if anyone of the process successfully, otherwise failed.
+
+When the process for the OCI image index completes successfully, the summary of the vulnerability report for the OCI image index is aggregated from the summaries of the artifacts referenced by the OCI image index. And the detailed vulnerability report is also aggregated from the detailed vulnerability reports of the artifacts referenced by the OCI image index and maybe possible redundancy.
```

Comment:
When an OCI image index is successfully scanned, the summary of the vulnerability report for the OCI image index is aggregated from the individual scan results of the the artifacts referenced by the index. The vulenrability report will show both sets of statistics.